lib/api/controllers/documentController.js

```diff
@@ -449,11 +478,12 @@ function doMultipleWriteActions (kuzzle, request, action) {
       modificationRequest.input.args.retryOnConflict = request.input.args.retryOnConflict;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 90 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)